lib/cheffish/base_resource.rb

```diff
@@ -1,7 +1,20 @@
 require 'chef_compat/resource'
+require 'cheffish/array_property'
 
 module Cheffish
   class BaseResource < ChefCompat::Resource
+    def initialize(*args)
+      super
+      chef_server run_context.cheffish.current_chef_server
```

Comment:
Would it make more sense to use a default or lazy default for this value rather than in the initialization?